.github/workflows/validate.yml

```diff
@@ -141,3 +141,21 @@ jobs:
       run: |
         . /etc/profile.d/lmod.sh
         tests/ci/run_build.py ohpc ${{ steps.files.outputs.added_modified }}
+
+  build_on_openEuler_2203:
```

Comment:
We should make sure `openEuler` x86_64 work first [1] (this will make the factory repo OK) , otherwise always failed.

[1] https://obs.openhpc.community/project/show/OpenHPC:3.0:Factory

Reply:
yes, not sure what is up with x86_64 but currently we cannot enable it